File: VSCode-C/Add.c
#include <stdio.h>

int main() {
    int numberOne, numberTwo, total;

    printf("Enter First Integer: ");
    scanf("%d", &numberOne);

    printf("Enter Second Integer: ");
    scanf("%d", &numberTwo);

    total = numberOne + numberTwo;
    printf("%d + %d = %d\n", numberOne, numberTwo, total);

    return 0;
}

File: VSCode-Java/Add.java
import java.util.Scanner;

public class Add {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter First Number: ");
        double firstNumber = sc.nextDouble();
        System.out.print("Enter Second Number: ");
        double secondNumber = sc.nextDouble();
        double total = firstNumber + secondNumber;
        System.out.println("The Total Equals: " + total);
    }
}
